packages/DropdownMenu/src/Trigger/Trigger.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+
+const handleKeyDown = e => {
+  e.stopPropagation();
+};
+
+const propTypes = {
+  isOpen: PropTypes.bool.isRequired,
+  handleOpenMenu: PropTypes.func.isRequired,
+  triggerRef: PropTypes.shape({ current: PropTypes.instanceOf(Object) }) || null,
+};
+
+const defaultProps = { triggerRef: null };
+
+const Trigger = props => {
+  const { isOpen, handleOpenMenu, triggerRef, ...otherProps } = props;
+  return (
+    <Button
+      ref={triggerRef}
+      aria-expanded={isOpen}
+      aria-haspopup="true"
+      onClick={handleOpenMenu}
```

Comment:
I think we should have `aria-controls` added here too.  It means we probably need to generate an `id` though... 😒 

Reference:
https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html